lib/v2/issuehunt/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'issuehunt.io': {
+        _name: 'Issue Hunt',
+        '.': [
+            {
+                title: '悬赏',
+                docs: 'https://docs.rsshub.app/programming.html#issue-hunt-xiang-mu-xuan-shang',
+                source: '/r/:username/:repo',
+                target: ({ username, repo }) => `/issuehunt/${username}/${repo}`,
```

Comment:
```suggestion
                target: ({ username, repo }) => `/issuehunt/funded/${username}/${repo}`,
```
The path should follow router.js